monero-outreach-docs/translations/ru/website-sections/guerrila_toolkit_ru.md

```diff
@@ -0,0 +1,65 @@
+2. Делитесь статьями и ссылками на ресурсы со своими друзьями и последователями в социальных сетях, даже на LinkedIn.
+3. Создавайте или принимайте участие в создании видео, подкастов, веб-сайтов и прочего онлайн контента.
+4. Звоните на радиостанции и говорите о Monero в прямом эфире.
+5. Напишите обзорную статью или письмо редактору вашей местной газеты и расскажите, как можно использовать Monero, чтобы сделать вашу локальную экономику более устойчивой к глобальным финансовым рискам.
+6. Свяжитесь с вашими местными предпринимателями и поговорите с ними о Monero. Используйте при этом инструменты, которые предлагает вам Monero Outreach. Мы продолжаем их разработку и совершенствование, так что регулярно проверяйте наличие обновлений.
+7. Комбинируете или дополняйте эти тактические приёмы так, как вам будет удобно. Внизу даются ссылки на источники, которые помогут вам в этом.
+
+## Активисты
+
+1. Активность в социальных сетях. Используйте highjack-хештеги как один из способов для привлечения внимания. Простой поисковик DuckDuckGo также подскажет вам несколько идей.
+2. Публичные мероприятия, посещение которых является прекрасным способом потратить ваше время и энергию. Подумайте о посещении концертов, ярмарок, больших конференций, политических событий и т. д.
+3. Мероприятия, посещаемые конкурентами. Применение тактики партизанского маркетинга на мероприятиях, на которых будут присутствовать ваши конкуренты, является более рискованным и противоречивым подходом, который может иметь значительные последствия. Подумайте о финансовых институтах и подобных организациях.
+4. Проецирование. При помощи обычного изображения, небольшого ноутбука и проектора вы можете добиться серьёзного воздействия. Как вариант можно использовать большой фонарик с наложенным на него трафаретом с логотипом Monero. Вспомните Бэтмена...
+5. Распространение информационных листовок. Массовая распечатка информации о Monero и распространение таких листовок на мероприятиях позволяет познакомить широкую (или новую) аудиторию с Monero.
+6. QR-коды. Массовая распечатка QR-кодов с небольшой рекламой, привлекающей внимание. Такие распечатки также можно распространять в местах проведения всевозможных мероприятий.
+7. Штемпелевание. Штемпелевание физических банкнот фиатных валют является эффективным и законным способом распространения информации о Monero. Согласно данным [stampmonero.com](https://stampmonero.com/), каждую долларовую банкноту, на которую вы поставите печать, могут увидеть 438 других человек.
+
+## Для дизайнеров
+
+1. [Monero Press Kit](https://www.getmonero.org/press-kit/) - здесь можно загрузить логотип Monero.
```

Comment:
We could add the Russian translation link for the getmonero.org site

https://www.getmonero.org/ru/press-kit/

[Monero Press Kit](https://www.getmonero.org/ru/press-kit/)

The same with the other links that go directly to the getmonero.org site.

Reply:
Fixed